qt/scientific_interfaces/Inelastic/Common/OutputPlotOptionsModel.h

```diff
@@ -18,42 +18,65 @@ using namespace MantidQt::Widgets::MplCpp;
 namespace MantidQt {
 namespace CustomInterfaces {
 
-class MANTIDQT_INELASTIC_DLL OutputPlotOptionsModel {
+class MANTIDQT_INELASTIC_DLL IOutputPlotOptionsModel {
 public:
-  OutputPlotOptionsModel(std::optional<std::map<std::string, std::string>> const &availableActions = std::nullopt);
-  /// Used by the unit tests so that m_plotter can be mocked
-  OutputPlotOptionsModel(ExternalPlotter *plotter,
+  virtual ~IOutputPlotOptionsModel() = default;
+
+  virtual bool setWorkspace(std::string const &workspaceName) = 0;
+  virtual void removeWorkspace() = 0;
+  virtual std::vector<std::string> getAllWorkspaceNames(std::vector<std::string> const &workspaceNames) const = 0;
+  virtual std::optional<std::string> workspace() const = 0;
```

Comment:
I tend to prefer this naming convention because it makes it clearer when a getter is accessing a member variable rather than something like `getAllWorkspaceNames` 
https://webkit.org/code-style-guidelines/#names-setter-getter